lib/commands/generate-examples.ts

```diff
@@ -24,6 +24,16 @@ export const builder: yargs.CommandBuilder = {
     describe: "the directory path contains payload.",
     string: true,
   },
+  c: {
+    alias: "config",
+    describe: "config path.",
```

Comment:
How about describe it as 'Readme config path'?

Reply:
Okay